schema/deploy/todo_data.sql

```diff
@@ -0,0 +1,15 @@
+-- Deploy todo:todo_data to pg
+-- requires: todo_app
+
+BEGIN;
+
+-- XXX Add DDLs here.
+create table if not exists todo_data (
```

Comment:
table should not exist, we would probably want sqitch to error out and roll back if it was the case